### ISSUE TITLE
Adding Open Interpreter to Devika

### ISSUE THREAD

**Issue by darrassi1** [stitionai/devika#269]
Integrating Open Interpreter with Devika in scientific research . Researchers could ask questions in natural language and receive insights from complex datasets, like identifying trends or generating hypotheses. This would improve data exploration and scientific discovery.
Open interpreter has internet acces

**Comment by virajkadam** [comment]
True, This option will boost the case and explore the power of Devika.
